bgpd/bgp_vty.c

```diff
@@ -7216,6 +7216,103 @@ DEFUN (clear_ip_bgp_all,
 	return bgp_clear_vty(vty, vrf, afi, safi, clr_sort, clr_type, clr_arg);
 }
 
+DEFUN (clear_ip_bgp_l2vpn_evpn_all,
```

Comment:
I see multiple issues here as explained below, I had fixed all of them locally and tested. I can get you the fix. 

But I have a bigger question. Why do we need to create a separate command altogether ? we can use existing clear ip bgp all command by adding l2vpn to afi and evpn to safi to the command options. Because this function seems to be exactly same as clear ip bgp all command function except couple of lines. 

Issues:
1) 
![image](https://user-images.githubusercontent.com/10162511/46828232-932b5080-cd4f-11e8-921f-ab3643d79223.png)

2) 
![image](https://user-images.githubusercontent.com/10162511/46828250-a2aa9980-cd4f-11e8-99d8-a988180e63c3.png)

I Fixed both the above and I have the fix.

Reply:
great that you can have the fix on separate pull request.
do you mind if I can see your pull request, then ?

about why I proposed to make a separate command for l2vpn evpn, this is because if you modify the main macros for afi and safi list, that means that a lot of other commands will be made available for l2vpn evpn. 
And I am not sure all the features are ready to work similarly with l2vpn evpn.

for adj-rib-in , I am sure of it.
but for the other features, I Am not sure of it.

Reply:
Either add new macros or remove the macros and put the afi/safi's we want and just use for soft clear right now imo.  @srimohans is correct here adding a new clear command is messy and going to cause problems down the line.

Reply:
ok. as this snipped of code has been done to help @srimohans, I let @srimohans give me feedback about what he needs - keep this pr or he will merge this and fix into his pr - 
thanks